src/cdk/listbox/listbox.ts

```diff
@@ -418,6 +418,19 @@ export class CdkListbox<T = unknown> implements AfterContentInit, OnDestroy, Con
   /** Whether the listbox currently has focus. */
   private _hasFocus = false;
 
+  /** A reference to the option that was active before the listbox lost focus. */
+  private _previousActiveItem: CdkOption<T> | null = null;
```

Comment:
Should we call this something like `_activeItemOnWindowBlur` so it's more obvious what it's used for.